bookdown/03-Acquisition_Software_Settings.Rmd

```diff
@@ -44,36 +46,39 @@ an account:
 
 [![Header of earthengine.google.com with link to create a GEE account
 highlighted in the red box to the
-right.](images/Screenshot 2023-10-25 at 11.56.29 AM.png)](https://earthengine.google.com)
+right.](images/EE_header.png)](https://earthengine.google.com)
 
 ### gcloud CLI
```

Comment:
Similar thought here as previous section. Is there a link with more info you could provide?